README.md

```diff
@@ -174,10 +187,25 @@ Run the following script to create a new entry that you will update with the sam
 ./scripts/update-changelog securedrop-foobar
 ```
 
-Finally, build the package by pointing to the tarball and package version:
+First verify the tarball you are about to package into a deb:
+
+```
+gpg --verify <package>.tar.gz.asc <package>.tar.gz
+```
+
+Build the package by pointing to the tarball and package version:
+
+```
+PKG_PATH=/path/to/package.tar.gz PKG_VERSION=x.y.z make securedrop-foobar
+```
+
+Save and publish your build logs to the `build-logs` repository, e.g. https://github.com/freedomofpress/build-logs/commit/786eb46672b07b5c635d87a075770b53a0ce3df9
+
+Open a PR to the `securedrop-debian-packages-lfs` repository with a test plan to verify the checksum in the build logs and (once appended to PR by a signature holder) that the new Release.gpg signature matches new Release file. The reviewer can perform verification by running:
 
 ```
-PKG_PATH=/path/to/tarball PKG_VERSION=0.x.y make securedrop-foobar
+sha256sum /path/to/built/package.deb
+gpg --verify repo/public/dists/xenial/Release.gpg repo/public/dists/xenial/Release
```

Comment:
If you elect to add shell expansion change above, should be added here too.

Reply:
gonna skip this